ipaserver/install/ipa_server_certinstall.py

```diff
@@ -134,14 +134,18 @@ def install_http_cert(self):
         dirname = certs.NSS_DIR
 
         old_cert = installutils.get_directive(paths.HTTPD_NSS_CONF,
-                                              'NSSNickname')
+                                              'NSSNickname', quote_char="'")
```

Comment:
Why is it safe to assume the `NSSNickname` will always be enclosed in single quotes?

mod_nss allows values to be enclosed in double quotes. If double quotes were indeed used, they'd be interpreted as a part of the value, which is not intended. It would be the exact same issue as ticket 6460, except for double quotes instead of single quotes.